swivel/nearest.py

```diff
@@ -55,21 +56,21 @@
   elif len(parts) == 3:
     vs = [vecs.lookup(w) for w in parts]
     if any(v is None for v in vs):
-      print 'not in vocabulary: %s' % (
-          ', '.join(tok for tok, v in zip(parts, vs) if v is None))
+      print('not in vocabulary: %s' % (
+            ', '.join(tok for tok, v in zip(parts, vs) if v is None)))
 
       continue
 
     res = vecs.neighbors(vs[2] - vs[0] + vs[1])
 
   else:
-    print 'use a single word to query neighbors, or three words for analogy'
+    print('use a single word to query neighbors, or three words for analogy')
     continue
 
   if not res:
     continue
 
   for word, sim in res[:20]:
-    print '%0.4f: %s' % (sim, word)
+    print('%0.4f: %s' % (sim, word))
 
-  print
+  print('')
```

Comment:
`print()` will print `()` in Python 2 unless you `from __future__ import print_function`